ee/server/service/certificate_authorities.go

```diff
@@ -163,6 +144,40 @@ func (svc *Service) NewCertificateAuthority(ctx context.Context, p fleet.Certifi
 	return createdCA, nil
 }
 
+func (svc *Service) validatePayload(p *fleet.CertificateAuthorityPayload, errPrefix string) error {
+	casToCreate := 0
+	if p.DigiCert != nil {
+		casToCreate++
+	}
+	if p.Hydrant != nil {
+		casToCreate++
+	}
+	if p.NDESSCEPProxy != nil {
+		casToCreate++
+	}
+	if p.CustomSCEPProxy != nil {
+		casToCreate++
+	}
+	if casToCreate == 0 {
+		return &fleet.BadRequestError{Message: fmt.Sprintf("%sA certificate authority must be specified", errPrefix)}
+	}
+	if casToCreate > 1 {
+		// handle showing this error only for create and update at the moment. If more cases are needed then we
+		// should probably pass in the verb instead of checking the errPrefix
+		var verb string
+		if strings.Contains(errPrefix, "create") {
```

Comment:
Would this need to be `add` ?

The current errPrefix defined in New Cert Auth is: `errPrefix := "Couldn't add certificate authority. "`

Reply:
i can remove this code as this is only used for created CAs now. I went down a path where I was gonna try to use this for created and updated but decided against it.